shoop/core/management/commands/shoop_extractmessages.py

```diff
@@ -0,0 +1,72 @@
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2015, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+"""
```

Comment:
we need to have 1 script that does the extracting, script under _misc doesn't extract everything, this combined extracting does. Management command feels more like "part of shoop" and gives more tools to tie the extraction into shoop only, and skip the other apps. The script under _misc extracts (if wanted) from everywhere and compiles them to one shoop-static.pot, which, in my opinion isn't what we need.